pymatgen/core/lattice.py

```diff
@@ -1052,6 +1052,8 @@ def get_points_in_sphere(
     ) -> Union[
         List[Tuple[np.ndarray, float, int, np.ndarray]],
         Tuple[List[np.ndarray], List[float], List[int], List[np.ndarray]],
+        List[Any],
```

Comment:
Why [Any] here?

Reply:
Just to accommodate the case where no neighbors are found, in which I simply return an empty list or a tuple of empty lists. Otherwise, I will need to design an empty return that suits the return format.

Reply:
Would `Optional` not be appropriate here?

Reply:
Let me check. Thanks